pkg/controller/plan/kubevirt.go

```diff
@@ -641,6 +660,83 @@ func (r *KubeVirt) EnsureDataVolumes(vm *plan.VMStatus, dataVolumes []cdi.DataVo
 	return
 }
 
+func (r *KubeVirt) vddkConfigMap(labels map[string]string) (*core.ConfigMap, error) {
+	data := make(map[string]string)
+	if r.Source.Provider.UseVddkAioOptimization() {
+		buffSize := r.Source.Provider.Spec.Settings[api.VddkAioBufSize]
+		if buffSize == "" {
+			buffSize = VddkAioBufSizeDefault
+		}
+		buffCount := r.Source.Provider.Spec.Settings[api.VddkAioBufCount]
+		if buffCount == "" {
+			buffCount = VddkAioBufCountDefault
+		}
+		data["vddk-config-file"] = fmt.Sprintf(
```

Comment:
Should we make this the official ConfigMap key name? In the CDI pull request, I mount the ConfigMap as a volume and then look for the first file in that volume, but it seems like it would be less trouble to just look for this fixed name.